travis/push-to-pypi.sh

```diff
@@ -45,11 +49,17 @@ elif [ -n "$TEST_PYPI_RELEASE" ]; then
         sed -i "s/${UPSTREAM_IMAGE_TAG}.*$/${UPSTREAM_IMAGE_Z_TAG}/" acc_provision/versions.yaml
         python setup.py sdist
         twine upload --repository testpypi -u ${TEST_PYPI_USER} -p ${TEST_PYPI_PASS} dist/$DEV_WHEEL_NAME
-        $SCRIPTS_DIR/push-to-cicd-status.sh "https://test.pypi.org/project/acc-provision/"${OVERRIDE_VERSION}"/#files" "${DEV_TAG_NAME}"
+        if [ $? -ne 0 ]; then
+            TWINE_UPLOAD="false"
+        fi
```

Comment:
should we call the update shell script if TWINE fails ? or we just bail out? why to process further if twine is failing

Reply:
If it is a release or rc job, container images artifacts can be updated. for example if a job with rc tag is run and some container image with the right tag doesn't exist when the job is run. once it is available, we can re run the travis job to update container image data.

Reply:
ok this is to make idempotent in case it run for multiple times for same tag so don't error out but to converge . Sure, I think for next phase we can add exit for at least docker pushes if that fails . I understand in twine we can do only a single push at once.